### PR TITLE
feat: add comments add component

### PR DESCRIPTION
**Pull request by dyersituations**
https://kiva.atlassian.net/browse/ACK-924

- Created component for adding a comment to an activity feed
- Added it to the comments container
- Removed the activity ID, since the app using these components will be interacting with the activity

image

image

### CODE REVIEW

File: @kiva/kv-components/vue/KvCommentsAdd.vue [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+						data-hj-suppress
+						tw-inline-block
+						tw-w-3.5
+						tw-h-3.5
+						tw-rounded-full
+						tw-overflow-hidden
+						tw-object-fill
+					"
+				>
+				<div class="tw-font-medium">
```
Comment (mcstover) [inline @kiva/kv-components/vue/KvCommentsAdd.vue:19]: Add `data-hj-suppress`

File: @kiva/kv-components/vue/KvCommentsAdd.vue [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+					{{ userDisplayName }}
+				</div>
+			</div>
+			<kv-text-input
+				:id="ADD_COMMENT_ID"
+				v-model="addCommentValue"
+				placeholder="Add a comment to this loan..."
+				class="tw-grow"
```
Comment (mcstover) [inline @kiva/kv-components/vue/KvCommentsAdd.vue:27]: Add `data-hj-suppress` here

File: @kiva/kv-components/vue/KvCommentsAdd.vue [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+			commentButtonState,
+			cancel,
+			comment,
+		};
+	},
+};
+</script>
+
+<style lang="postcss" scoped>
+>>> input {
```
Comment (mcstover) [inline @kiva/kv-components/vue/KvCommentsAdd.vue:104]: Can we use the deep() syntax here instead?

File: @kiva/kv-components/vue/KvCommentsAdd.vue (hunk 1 above)
Comment (dyersituations) [reply @kiva/kv-components/vue/KvCommentsAdd.vue]: Ah yup, good call, done.

File: @kiva/kv-components/vue/KvCommentsAdd.vue (hunk 2 above)
Comment (dyersituations) [reply @kiva/kv-components/vue/KvCommentsAdd.vue]: Done.

File: @kiva/kv-components/vue/KvCommentsAdd.vue (hunk 3 above)
Comment (dyersituations) [reply @kiva/kv-components/vue/KvCommentsAdd.vue:104]: Nope, not supported currently in this project (not true Vue3).

**Review by mcstover — APPROVED** [review]
Thanks for the tweaks. Not sure about the css deep options in this repo...

**Pull request merged**